operations/iscsi_sbps/iscsi_workers.md

```diff
@@ -0,0 +1,61 @@
+In CSM 1.7.0, selective node personalization is supported where user/admin can limit/configure the required worker
+nodes as iSCSI SBPS targets. This is done by creating HSM group named `iscsi_worker` and add worker node xnames to
+this group which are required to be configured as iSCSI targets. See ‘Steps to create HSM group’ section of [HSM groups](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/iscsi_sbps/HSM_groups_iscsi.md)
+
+If user/admin is not intended to limit the worker nodes for iSCSI SBPS i.e no `iscsi_worker` group is created, then all
+worker nodes will be configured as iSCSI targets. If the user does not want to configure any of the worker nodes as iSCSI targets, then the HSM group `iscsi_worker` can be kept empty which is not suggestible as no compute nodes can be booted.
+
+This group needs to be created at certain stages during CSM install/upgrade:
+
+During fresh install of CSM 1.7.0: `iscsi_worker` HSM group needs to be created before the `Configure management nodes with CFS` step of [CSM 1.7.0 installation](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/install/configure_administrative_access.md#8-configure-management-nodes-with-cfs)
+
+During upgrade of CSM 1.7.0: `iscsi_worker` HSM group needs to be created before ‘management nodes rollout’ step during CSM upgrade using IUF. See [CSM 1.7.0 Upgrade](https://github.com/Cray-HPE/docs-csm/blob/ae9482ae85bc2eede3ac61079ffed5a00fb0a126/operations/iuf/workflows/management_rollout.md)
+
+Post install/upgrade if it is required to add any new worker node to the cluster or need to remove an existing worker node from the cluster, then see the [add/remove worker node](https://github.com/Cray-HPE/docs-csm/blob/release/1.7/operations/node_management/Add_Remove_Replace_NCNs/Add_Remove_Replace_NCNs.md#add-worker-storage-or-master-ncns).
+
+Post this, if it is required to add/remove any worker node to/from `iscsi_worker` group, then new node can be added or removed as below:
+
+Adding new node to `iscsi_worker` group:
+
+```bash
+cray hsm groups members create --id <xname_of_worker_node> iscsi_worker
+```
+
+e.g
+
+```bash
+# cray hsm groups members create --id  x3000c0s18b0n0 iscsi_worker
+```
+
+e.g
+
+```text
+[[results]]
+URI = "/hsm/v2/groups/iscsi_worker/members/x3000c0s18b0n0"
+```
+
+Deleting the worker node from `iscsi_worker` group:
+
+```bash
+# cray hsm groups members delete <xname> iscsi_worker
+```
+
+e.g.
+
+```text
+# cray hsm groups members delete x3000c0s29b0n0 iscsi_worker
+
+ncn-m001:~ # cray hsm groups members delete x3000c0s29b0n0 iscsi_worker
+code = 0
+message = "deleted 1 entry"
+```
+
+Then re-run iSCSI CFS layer on worker nodes using below script:
+`/usr/share/doc/csm/scripts/operations/configuration/refresh_worker_iscsi_config.py`
+
+Also, the latest CSM documentation RPM must be installed on the node where the
+procedure is being performed. See `CFS configuration and component update`
```

Comment:
This should be a link to the "Check for documentation updates", not to the HSM groups page.

Reply:
The title of this link and the link itself are not right. This entire step should basically mirror the format of [this one from the Reboot NCNs procedure](https://github.com/Cray-HPE/docs-csm/blob/0da98916bd98dfbc348ecea24d9f1e5cf59ed7a0/operations/node_management/Reboot_NCNs_manual.md?plain=1#L327-L334)